src/__tests__/__files__/pages/test-global-scss.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Alert from 'shared-ui/components/Alert';
+
+import Box from '../components/Box';
+
+const HomePage = () => {
+  return (
+    <main>
+      <Alert>My alert</Alert>
```

Comment:
Aren't you missing the Textarea component here?

Reply:
Yes, indeed!

Reply:
Tests should now succeed with 8fadc6a